openshift/osdctl
# osd-11717 Onv command for osdctl

Adds "osdctl network verify-egress" subcommand which integrates with osd-network-verifier

 
To test locally,

Clone this branch.
Build osdctl binary by running make build.
Select a subnet and get AWS credentials for it.

run AWS_ACCESS_KEY_ID=XXX AWS_SECRET_ACCESS_KEY=XXXX AWS_SESSION_TOKEN=XXX . /dist/<select dist for your os>/osdctl network verify-egress --subnet-id=<subnet-id> --region=<region>
